spec/controllers/v2/value_definitions_controller_spec.rb

```diff
@@ -0,0 +1,157 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe V2::ValueDefinitionsController do
+  let(:attributes) do
+    {
+      ref_id: :ref_id,
+      title: :title,
+      description: :description,
+      value_type: :value_type,
+      default_value: :default_value
+    }
+  end
+
+  context 'With user' do
+    let(:current_user) { FactoryBot.create(:v2_user) }
+
+    before(:each) do
```

Comment:
No need for the `each` as it is implicit:
```suggestion
    before do
```